package.json

```diff
@@ -1,113 +1,117 @@
-  "engines": {
-    "vscode": "^1.91.0"
-  },
-  "icon": "logo.png",
-  "categories": [
-    "Other"
-  ],
-  "main": "./out/extension.js",
-  "activationEvents": [],
-  "contributes": {
-    "commands": [
-      {
-        "command": "extension.filesToMarkdown",
-        "title": "Copy for Context"
-      },
-      {
-        "command": "extension.tabsToMarkdown",
-        "title": "Copy All Open Tabs for Context"
-      },
-      {
-        "command": "extension.singleFileToMarkdown",
-        "title": "Copy This File for Context"
-      },
-      {
-        "command": "extension.tabGroupToMarkdown",
-        "title": "Copy This Tab Group for Context"
-      }
-    ],
-    "menus": {
-      "explorer/context": [
-        {
-          "command": "extension.filesToMarkdown"
-        }
-      ],
-      "editor/title": [
-        {
-          "command": "extension.tabsToMarkdown"
-        }
-      ],
-      "editor/title/context": [
-        {
-          "command": "extension.singleFileToMarkdown",
-          "group": "navigation@999",
-          "when": "resource"
-        },
-        {
-          "command": "extension.tabGroupToMarkdown",
-          "group": "navigation@999"
-        },
-        {
-          "command": "extension.tabsToMarkdown",
-          "group": "navigation@999"
-        }
-      ],
-      "commandPalette": [
-        {
-          "command": "extension.filesToMarkdown",
-          "title": "Copy for Context"
-        },
-        {
-          "command": "extension.tabsToMarkdown",
-          "title": "Copy All Open Tabs for Context"
-        },
-        {
-          "command": "extension.singleFileToMarkdown",
-          "title": "Copy This File for Context"
-        },
-        {
-          "command": "extension.tabGroupToMarkdown",
-          "title": "Copy This Tab Group for Context"
-        }
-      ]
-    }
-  },
-  "scripts": {
-    "vscode:prepublish": "npm run compile",
-    "compile": "tsc -p ./",
-    "watch": "tsc -watch -p ./",
-    "pretest": "npm run compile && npm run lint",
-    "lint": "eslint 'src/**/*.{js,ts,tsx}' --quiet",
-    "lint:fix": "eslint 'src/**/*.{js,ts,tsx}' --quiet --fix",
-    "format": "prettier --write 'src/**/*.{js,ts,tsx}'",
-    "format:check": "prettier --check 'src/**/*.{js,ts,tsx}'",
-    "test": "node ./out/test/runTest.js",
-    "package": "vsce package",
-    "cpi": "npm run compile && npm run package && code-insiders --install-extension $(ls context-copy-*.vsix | head -n 1)"
-  },
-  "devDependencies": {
-    "@types/mocha": "^10.0.7",
-    "@types/node": "20.x",
-    "@types/vscode": "^1.91.0",
-    "@typescript-eslint/eslint-plugin": "^7.14.1",
-    "@typescript-eslint/parser": "^7.11.0",
-    "@vscode/test-cli": "^0.0.10",
-    "eslint": "^8.57.0",
-    "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-prettier": "^5.2.1",
-    "husky": "^9.1.4",
-    "lint-staged": "^15.2.9",
-    "prettier": "^3.3.3",
-    "typescript": "^5.4.5"
-  }
-}
\ No newline at end of file
+{
```

Comment:
Can you make the diff for this file smaller please?

Reply:
idk what the issue is but I cant get the diff to work properly here, I've copy pasted the files before and after here you can check it out https://www.diffchecker.com/cX1NvPtY/